tessera-dist/src/main/java/com/quorum/tessera/launcher/Launcher.java

```diff
@@ -116,8 +119,12 @@ public List<TesseraServer> launchServer(final Config config) throws Exception {
       recoveryServer.start();
       LOGGER.debug("Started recovery server");
 
-      LOGGER.info("Waiting for nodes to synchronise with peers");
-      Thread.sleep(10000);
+      final var waitTimeBeforeRecoveryStartsInMillis =
+          intervalPropertyHelper.partyInfoInterval() * 2L;
```

Comment:
this portion is grey, party info of another node might be scheduled at a different interval from your own tessera

Reply:
That is correct.